drafts/draft-ioametal-ippm-6man-ioam-ipv6-deployment.xml

```diff
@@ -218,42 +218,44 @@
           fields is the same.</t>
 
           <t hangText="C2">Given that IOAM data fields increase the total size
-          of the packet, the packet size could exceed the PMTU. In particular,
-          the incremental trace IOAM HbH Option, which is proposed to support
-          hardware implementations of IOAM, changes Option Data Length
-          en-route. Operators of an IOAM domain are to ensure that the
-          addition of OAM information does not lead to fragmentation of the
-          packet. Careful control of the MTU in a network is one of the
-          reasons why IOAM is considered a domain specific feature, see also
-          <xref target="I-D.ietf-ippm-ioam-data"/>. In addition, the PMTU
+          of the packet, the size of the packet including the IOAM data could 
+		  exceed the PMTU. In particular, the incremental trace IOAM HbH 
+		  Option, which is proposed to support hardware implementations of 
+		  IOAM, changes Option Data Length en-route. Operators of an IOAM 
+		  domain are to ensure that the addition of OAM information does not 
+		  lead to fragmentation of the packet, by configuring the MTU of 
+		  transit routers and switches to a sufficiently high value. Careful 
```

Comment:
This is just one way to not run into issues with MTU size. Should we rather say "e.g. by configuring the MTU..." - because there are other ways to address this issue (e.g. configure IOAM in a way that MTU cannot be exceeded).